### PR TITLE
Report check time

### PR DESCRIPTION
**Pull request by joakim-hove**
Rebased++ version of #1710

On top of #1729

### CODE REVIEW

File: src/opm/output/eclipse/EclipseIO.cpp [hunk 1]
```diff
@@ -263,7 +263,7 @@ void EclipseIO::writeTimeStep(const SummaryState& st,
     if (!isSubstep) {
         for (const auto& report : schedule.report_config(report_step)) {
             std::stringstream ss;
-            RptIO::write_report(ss, report.first, report.second, schedule, report_step);
+            RptIO::write_report(std::cout, report.first, report.second, schedule, report_step);
```
Comment (bska) [inline src/opm/output/eclipse/EclipseIO.cpp:266]: Yeah, no.

Comment (joakim-hove) [reply src/opm/output/eclipse/EclipseIO.cpp:266]: Relax my friend - this is work in progress

Comment (bska) [reply src/opm/output/eclipse/EclipseIO.cpp:266]: > this is work in progress

Then please mark it as such.

**Comment by joakim-hove** [comment]
OK - now this should be ready to be merged. In this PR the WELSPECS report will actually go to the PRT file. Some of fields are still missing - that is being worked on.

Comments?

**Comment by joakim-hove** [comment]
jenkins build this please

**Comment by bska** [comment]
> > Do you really need all of the `std::endl` calls?
> 
> I like `std::endl` - but sure I can change that.

Thanks.  It's mostly that I don't see the that the benefit of regularly calling `ostream::flush()` outweighs the additional cost.

**Pull request merged**